test/Driver/EDMF/closures/mixing_length.jl

```diff
@@ -0,0 +1,119 @@
+#### Mixing length model kernels
+
+function mixing_length(
+    ss::SingleStack{FT, N},
+    m::MixingLengthModel,
+    source::Vars,
+    state::Vars,
+    diffusive::Vars,
+    aux::Vars,
+    t::Real,
+    direction,
+    δ::FT,
+    εt::FT,
+) where {FT, N}
+
+
+    # Q - do I need to define L as a vector ?
+    # need to code the functions: obukhov_length, ustar, ϕ_m, lamb_smooth_minimum
+
+    # Alias convention:
+    gm = state
+    en = state
+    up = state.edmf.updraft
+    gm_s = source
+    en_s = source
+    up_s = source.edmf.updraft
+    en_d = state.edmf.environment.diffusive
+
+    z = gm_a.z
+    # Parameters
+    cv_d::FT = cv_d(param_set)
+    cv_v::FT = cp_v(param_set)
+    cv_l::FT = cp_l(param_set)
+    cv_i::FT = cp_i(param_set)
+    T0::FT = T_0(param_set)
+    e_int_i0::FT = e_int_i0(param_set)
+    ϵ_v = 1. / molmass_ratio(param_set)
```

Comment:
```suggestion
    ϵ_v ::FT = 1 / molmass_ratio(param_set)
```
The `1.0` promote `ϵ_v` to Float64 (even if `FT(molmass_ratio(param_set))` is `Float32`)